src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2504,4 +2492,74 @@ Uninstall the following packages: {0}?</value>
   <data name="GetStartedGuideSurveyTitle" xml:space="preserve">
     <value>Finished</value>
   </data>
+  <data name="PackageStatePendingUnload" xml:space="preserve">
+    <value>Scheduled to be unloaded</value>
+  </data>
+  <data name="PackageStateError" xml:space="preserve">
+    <value>Error</value>
+  </data>
+  <data name="PackageStateLoaded" xml:space="preserve">
+    <value>Loaded</value>
+  </data>
+  <data name="PackageStateUnloaded" xml:space="preserve">
+    <value>Unloaded</value>
+  </data>
+  <data name="PackageStateScheduledForDeletion" xml:space="preserve">
+    <value>Scheduled for Delete</value>
+  </data>
+  <data name="PackageStateScheduledForUnload" xml:space="preserve">
+    <value>Scheduled for Unload</value>
+  </data>
+  <data name="PackageStateScheduledForLoad" xml:space="preserve">
+    <value>Scheduled for Load</value>
+  </data>
+  <data name="PackageStateErrorTooltip" xml:space="preserve">
+    <value>Error.
+This package has not been loaded due to the following error:
+{0}.</value>
+  </data>
+  <data name="PackageStateLoadedTooltip" xml:space="preserve">
+    <value>Loaded.
+This package is loaded and ready to be used.</value>
+  </data>
+  <data name="PackageStateUnloadedTooltip" xml:space="preserve">
+    <value>Unloaded.
+This package has not been loaded because another conflicting package was loaded before it.</value>
+  </data>
+  <data name="PackageStateScheduledForUnloadTooltip" xml:space="preserve">
+    <value>Scheduled for Unload.
+This package will be unloaded after the next Dynamo restart.</value>
+  </data>
+  <data name="PackageStateScheduledForLoadTooltip" xml:space="preserve">
+    <value>Scheduled for Load.
```

Comment:
@jasonstratton what is this state for again? Can't packages be loaded immediately? Why do they need to be scheduled to load?

Reply:
I guess the scenario is when Immediate Loading cannot happen because of conflict with an existing Package (that in turn cannot be deleted immediately)